sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -37,6 +37,7 @@
 
 from google.cloud import storage
 from google.cloud.exceptions import NotFound
+from google.cloud.storage.fileio import DEFAULT_RETRY
```

Comment:
Recommend:
from google.cloud.storage.retry import DEFAULT_RETRY

fileio does have DEFAULT_RETRY but only because it imports it from the source, above, so it's not intentionally exporting it.

Reply:
Done.